Codeinwp/neve
# feat: Menu icon variants

### Summary
Added a new control for Menu Icon component in HFG.
Inside the Style tab of the Menu Icon component, a new control is available that will allow the user to switch the icon used.

If an icon other than default is being used, some extra css will be inlined and minified for the icon to work as expected.

### Will affect visual aspect of the product

YES

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/156396430-2dba9438-be35-4020-95e6-94dbb434bc83.png)


### Test instructions


1. Add the Menu Icon component or use the one inside the Header Mobile.
2. Change the Icon style to other than default.
3. Check that all styles work and also that the default style works as before.


Closes #992.
